scripts/sysman/engine.yml

```diff
@@ -27,22 +27,24 @@ etors:
       desc: "Access information about a single compute engine - this is an engine that can process compute kernels. Note that single engines may share the same underlying accelerator resources as other engines so activity of such an engine may not be indicative of the underlying resource utilization - use $S_ENGINE_GROUP_3D_RENDER_COMPUTE_ALL for that."
     - name: RENDER_SINGLE
       desc: "Access information about a single render engine - this is an engine that can process both 3D content and compute kernels. Note that single engines may share the same underlying accelerator resources as other engines so activity of such an engine may not be indicative of the underlying resource utilization - use $S_ENGINE_GROUP_3D_RENDER_COMPUTE_ALL for that."
+    - name: MEDIA_CODEC_SINGLE
+      desc: "Access information about a single media  engine. Note that single engines may share the same underlying accelerator resources as other engines so activity of such an engine may not be indicative of the underlying resource utilization - use $S_ENGINE_GROUP_MEDIA_ALL for that."
     - name: MEDIA_DECODE_SINGLE
-      desc: "Access information about a single media decode engine. Note that single engines may share the same underlying accelerator resources as other engines so activity of such an engine may not be indicative of the underlying resource utilization - use $S_ENGINE_GROUP_MEDIA_ALL for that."
+      desc: "1.6": "Note: This enum is deprecated. Access information about a single media decode engine. Note that single engines may share the same underlying accelerator resources as other engines so activity of such an engine may not be indicative of the underlying resource utilization - use $S_ENGINE_GROUP_MEDIA_ALL for that."
     - name: MEDIA_ENCODE_SINGLE
-      desc: "Access information about a single media encode engine. Note that single engines may share the same underlying accelerator resources as other engines so activity of such an engine may not be indicative of the underlying resource utilization - use $S_ENGINE_GROUP_MEDIA_ALL for that."
+      desc: "1.6": "Note: This enum is deprecated. Note that single engines may share the same underlying accelerator resources as other engines so activity of such an engine may not be indicative of the underlying resource utilization - use $S_ENGINE_GROUP_MEDIA_ALL for that."
```

Comment:
thanks @sanilkumar0 . We need two versions of the desc, one for 1.0 and one for 1.5. as is right now, we wouldn't have any description for MEDIA_DECODE_SINGLE or  MEDIA_ENCODE_SINGLE for specs with versions less than 1.6

Reply:
Changes committed to reflect on the comment's

Reply:
It's a bit tautological to have "Note: ... . Note that".  I would drop "Note that" from all of these v1.6 texts when "Note:" is added before it.

Reply:
Fixed